hw_3/t1.py

```diff
@@ -0,0 +1,13 @@
+import os
+os.system('cls')
+
+import random
+n = int(input("Введите количество сгенерированных чисел: "))
+arr = [random.randint(0, 6) for _ in range(n)]
+print(arr)
+X = int(input("Введите проверяемое число: "))
+count = 0
+for i in range(n):
+    if arr[i] == X:
```

Comment:
Не обязательно использовать функцию `range`, можно проходиться напрямую по элементам списка.
```suggestion
for i in arr:
    if i == X:
```

Reply:
Благодарю!